Source/WebCore/animation/ViewTimeline.h

```diff
@@ -42,12 +42,6 @@ class Element;
 
 struct TimelineRange;
 
-struct ViewTimelineInsets {
-    std::optional<Length> start;
-    std::optional<Length> end;
-    bool operator==(const ViewTimelineInsets&) const = default;
-};
```

Comment:
Why is that change being made?

Reply:
Moved build fixes to https://github.com/WebKit/WebKit/pull/39640.